demo/test.c

```diff
@@ -2313,7 +2313,7 @@ static int test_mp_read_write_ubin(void)
 
    size = mp_ubin_size(&a);
    printf("mp_to_ubin_size  %zu\n", size);
-   buf = malloc(sizeof(*buf) * size);
+   buf = (unsigned char *)malloc(sizeof(*buf) * size);
```

Comment:
uint8_t

Reply:
Disregard this on 1.x

Reply:
> uint8_t

No, uint8_t is wrong here, because mp_to_ubin has an "unsigned char *" argument. (on develop it might be different, but this PR is for support/1.x)